nilearn/plotting/surface/surf_plotting.py

```diff
@@ -921,7 +918,7 @@ def plot_surf_roi(
         When using matplotlib as engine,
         `avg_method` will default to ``"median"`` if ``None`` is passed.
 
-    threshold : a number or None, default=1e-14
+    threshold : :obj:`int` or :obj:`float`, None, default=1e-14
```

Comment:
Could we set default value for `threshold` as `None` for this function?

Reply:
and have the old default as a fall back if None is passed?

Reply:
Not really, in other functions in the module, threshold defaults to None, and the image is not thresholded. 

For `plot_surf_roi`, docstring says:
```
        Threshold regions that are labeled 0.
        If you want to use 0 as a label, set threshold to None.
```

So does that mean by default for roi the regions labeled 0 should be thresholded?

Reply:
checked here the values used by the only surface atlas we 'ship':

https://nilearn.github.io/dev/modules/generated_reports/masker_reports_examples.html#surface-maskers

no unknown regions label and no vertex with a 0 value

BIDS does not have a label for unknown regions but does recommend using 0 for 'background'

So I would set the value to None to be consistent and let users change it to what they want.

Reply:
@hndgzkn 
Will let you change the threshold default

Reply:
the tests pass: the docstring suggestion would be good IMO